ui/sections/containers/inspectors/docker-image-pull.reel/docker-image-pull.html

```diff
@@ -19,11 +20,12 @@
             "properties": {
                 "element": {"#": "inspector"},
                 "canDelete": false,
-                "canSave": false,
+                "canSave": true,
                 "canRevert": false,
                 "controller": {"@": "owner"}
             },
             "bindings": {
+                "canSave": {"<-": "!!@owner._selectedImage && !!@owner._selectedHost"},
```

Comment:
You should use `isSaveDisabled` to disable the Save button instead of hiding it.